src/main/resources/templates/oidcRedirect.html

```diff
@@ -14,120 +14,257 @@
-      xml:lang="en">
+<html xmlns:th="http://www.thymeleaf.org" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.thymeleaf.org " xml:lang="en">
+
 <head>
     <title>LTI Core Standard Launch OIDC Redirect (Behind the Scenes - Step 1)</title>
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
-    <link rel="stylesheet" type="text/css" th:href="@{/css/bootstrap.min.css}"/>
-    <link rel="stylesheet" type="text/css" th:href="@{/css/style.css}"/>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <link rel="stylesheet" type="text/css" th:href="@{/css/bootstrap.min.css}" />
+    <link rel="stylesheet" type="text/css" th:href="@{/css/style.css}" />
 </head>
+
 <body>
+<script type="text/javascript" th:src="@{/js/ltiPostMessage.js}"></script>
 
-<h4>LTI Core Standard Launch OIDC Redirect (Behind the Scenes - Step 1)</h4>
-<h5 class="alert alert-warning">In a real tool that is not for demonstration purposes, this step will be handled completely on the back end, so the user will never see this page/information as they will be sent directly to the tool's landing page (the next UI page).</h5>
-
-<button class="accordion large-font">
-    Click to view the form data we received from the platform
-</button>
-<div class="panel">
-    <div>
-        <ul class="list-group">
-            <li class="list-group-item list-group-item-primary">
-                <label>iss:</label>
-                <span th:text="${initiation_dto.iss}"/>
-            </li>
-            <li class="list-group-item list-group-item-primary">
-                <label>login_hint: </label>
-                <span th:text="${initiation_dto.loginHint}"/>
-            </li>
-            <li class="list-group-item list-group-item-primary">
-                <label>target_link_uri:</label>
-                <span th:text="${initiation_dto.targetLinkUri}"/>
-            </li>
-            <li class="list-group-item list-group-item-primary">
-                <label>lti_message_hint:</label>
-                <span th:text="${initiation_dto.ltiMessageHint}"/>
-            </li>
-            <li class="list-group-item list-group-item-primary">
-                <label>client_id (received):</label>
-                <span th:text="${client_id_received}"/>
-            </li>
-            <li class="list-group-item list-group-item-primary">
-                <label>lti_deployment_id (received):</label>
-                <span th:text="${deployment_id_received}"/>
-            </li>
-        </ul>
+    <h4>LTI Core Standard Launch OIDC Redirect (Behind the Scenes - Step 1)</h4>
+    <h5 class="alert alert-warning">In a real tool that is not for demonstration purposes, this step will be handled
+        completely on the back end, so the user will never see this page/information as they will be sent directly to
+        the tool's landing page (the next UI page).</h5>
```

Comment:
We should probably remove the parenthetical "(the next UI page)" since that's not true anymore with the cookieless solution.